File: TP2/Python/Punto4/Fede3p/Fede_Profesor.py
class Profesor:
    
    __nombre = ""
    __apellido = ""
    __materia = ""
    
    def __init__(self, nombre, apellido):
        self.__nombre = nombre
        self.__apellido = apellido
        
    def add_materia(self, materia):
        self.__materia.add(materia)
    
    def getNombre(self):
        return self.__nombre
    
    def set_nombre(self, nombre):
        self.__nombre = nombre
        
    def getApellido(self):
        return self.__apellido
    
    def set_apellido(self, apellido):
        self.__apellido = apellido
        
    def getMateria(self):
        return self.__materia
    
    def set_materia(self, materia):
        self.__materia = materia

File: TP3/JAVA/ejercicio 8/Fede3p/Fede_Profesor.java
public abstract class Fede_Profesor {
    
    private String nombre;
    private String apellido;
    private Integer edad;
    private Integer horasTrabajadas;

    public Fede_Profesor(String nombre, String apellido, Integer edad, Integer horasTrabajadas){
        this.nombre = nombre;
        this.apellido = apellido;
        this.edad = edad;
        this.horasTrabajadas = horasTrabajadas;
    }

    public abstract Double getRemuneracionMensual();

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public Integer getEdad() {
        return edad;
    }

    public void setEdad(Integer edad) {
        this.edad = edad;
    }

    public Integer getHorasTrabajadas() {
        return horasTrabajadas;
    }

    public void setHorasTrabajadas(Integer horasTrabajadas) {
        this.horasTrabajadas = horasTrabajadas;
    }

    
}
